README.md

```diff
@@ -71,44 +81,62 @@ To view the value contained in the secret as plain text:
 ```
 
-This command shows the secret information including the URI. After completing these steps, you should have a URI to a secret in an Azure Key Vault. Write this information down. You need it in a later step.
+This command shows the secret information including the URI. After completing these steps, you should have a URI to a secret in an Azure Key Vault. Copy the output from the previous command to Notepad. You need it in a later step.
 
-## Clone the Repo
+## Clone the repo
 
-Clone the repo in order to make a local copy for you to edit the source by running the following command:
+Run the following command to clone this Quickstart code to your local machine:
 
 ```
 git clone https://github.com/Azure-Samples/key-vault-node-quickstart.git
 ```
 
 ## Install dependencies
 
-Here we install the dependencies. Run the following commands
-    cd key-vault-node-quickstart
-    npm install
+Run the following command to install dependencies for "SDK version 3.0" and "SDK version 4.0":
+
+- SDK version 4.0
+
+```
+cd key-vault-node-quickstart-v4 
+```
+```
+npm install
+```
 
-This project used 2 node modules:
+- SDK version 3.0
 
-* [ms-rest-azure](https://www.npmjs.com/package/ms-rest-azure) 
-* [azure-keyvault](https://www.npmjs.com/package/azure-keyvault)
+```
+cd key-vault-node-quickstart-v3 
+```
+```
+npm install
+```
 
 ## Publish the web application to Azure
 
-Below are the few steps we need to do
+To publish this web application to Azure we need to create an Azure App Service, Azure Web App, and create a Deployment User.
 
-- The 1st step is to create a [Azure App Service](https://azure.microsoft.com/services/app-service/) Plan. You can store multiple web apps in this plan.
+1. Azure App Service
+
+The first step is to create an [Azure App Service](https://azure.microsoft.com/services/app-service/) Plan. You can store multiple web apps in this plan. Use the resource group that you created earlier in the following command:
 
     ```
-    az appservice plan create --name myAppServicePlan --resource-group myResourceGroup
+    az appservice plan create --name <myAppServicePlan> --resource-group <myResourceGroup>
     ```
-- Next we create a web app. In the following example, replace <app_name> with a globally unique app name (valid characters are a-z, 0-9, and -). The runtime is set to NODE|6.9. To see all supported runtimes, run az webapp list-runtimes
+
+2. Azure Web App
+
+Next we create a web app. In the following example, replace <AppName> with a globally unique app name (valid characters are a-z, 0-9, and -). The runtime is set to NODE|6.9. To see all supported runtimes, run az webapp list-runtimes:
+    
     ```
     # Bash
-    az webapp create --resource-group myResourceGroup --plan myAppServicePlan --name <app_name> --runtime "NODE|6.9" --deployment-local-git
+    az webapp create --resource-group <myResourceGroup> --plan <myAppServicePlan> --name <AppName> --runtime "NODE|6.9" --deployment-local-git
```

Comment:
If you are going to prefix some with "my", then you should prefix them all with "my".  I recommend not using "my", but that is personal preference.

Reply:
all use "My"